rllib/algorithms/algorithm.py

```diff
@@ -23,14 +26,10 @@
     Union,
 )
 
-import gym
-import numpy as np
-import pkg_resources
-from packaging import version
-
 import ray
 from ray._private.usage.usage_lib import TagKey, record_extra_usage_tag
 from ray.actor import ActorHandle
+import ray.cloudpickle as pickle
```

Comment:
From the cloudpickle repo:
> Using cloudpickle for long-term object storage is not supported and strongly discouraged.

Just as a reminder that, in the long run, we should checkpoint with cloudpickle.